python/test/unit/language/test_core.py

```diff
@@ -3985,16 +3985,8 @@ def dot_scale_kernel(a_base, stride_a0, stride_a1, a_scale, b_base, stride_b0, s
         tl.store(out_ptr, c.to(tl.bfloat16))
 
     @triton.jit
-    def mxfp_upcast_kernel(
-        x_ptr,
-        scale_ptr,
-        mxfp_ptr,
-        N,
-        e_bits: tl.constexpr,
-        m_bits: tl.constexpr,
-        to_type: tl.constexpr,
-        BLOCK_SIZE: tl.constexpr,
-    ):
+    def mxfp_upcast_kernel(x_ptr, scale_ptr, mxfp_ptr, N, e_bits: tl.constexpr, m_bits: tl.constexpr,
+                           to_type: tl.constexpr, BLOCK_SIZE: tl.constexpr, is_xpu: tl.constexpr):
```

Comment:
please also re-clang format, that way is more similar to upstream.
```suggestion
                           to_type: tl.constexpr, BLOCK_SIZE: tl.constexpr, is_xpu: tl.constexpr,):
```